docs/src/snap/reference/troubleshooting.md

```diff
@@ -82,8 +82,31 @@ EOF
 reboot
 ```
 
+## The path required for the containerd socket already exists
+
+### Problem
+
+{{product}} tries to create the containerd socket to manage containers,
+but it fails because the socket file already exists, which indicates another
+installation of containerd on the system.
+
+### Explanation
+
+In classic confinment mode, {{product}} uses the default containerd
+paths. This means that a {{product}} installation will conflict with
+any existing system configuration where containerd is already installed.
+For example, if you have Docker installed, or another Kubernetes distribution
+that uses containerd.
+
+### Solution
+
+We recommend running {{product}} in an isolated environment, for this purpose,
+you can create a LXD container for your installation. See
+[Install {{product}} in LXD][lxd-install] for instructions.
```

Comment:
Good idea to link to isolated env instead

Reply:
Wdym?

Reply:
I just meant this is a better solution than asking the user to uninstall/clean up existing deployments